### DISCUSSION TITLE
A new intergrated yank/clipboard tool: smarkyank.nvim.

### DISCUSSION THREAD

**Discussion by charliie-dev** [ayamir/nvimdots#275] [Ideas]
I was blown away by this fresh hot plugin! This plugin intergrates register/system clipboard/tmux/ssh(osc52) to make copy&pasting on nvim better, which was really a pain in the ass before. I remember spending hours and hours of time trying to make things work properly, what a nightmare.
I think this plugin would be really nice to place in this config.
https://github.com/ibhagwan/smartyank.nvim

**Comment by ayamir** [comment]
Maybe you can use it for your own, I use [copyq](https://github.com/hluk/CopyQ/) to manage system-wide clipboard.

**Reply by charliie-dev** [reply]
That looks cool! Thanks!
